pkg/operators/localmanager/localmanager.go

```diff
@@ -497,6 +530,27 @@ func (l *localManager) InstantiateDataOperator(gadgetCtx operators.GadgetContext
 		return nil, err
 	}
 
+	containersDs, err := gadgetCtx.RegisterDataSource(datasource.TypeSingle, "containers")
+	if err != nil {
+		return nil, fmt.Errorf("creating datasource: %w", err)
+	}
+	containersDs.AddAnnotation("cli.default-output-mode", "none")
+
+	eventTypeField, err := containersDs.AddField("event_type", api.Kind_String)
+	if err != nil {
+		return nil, fmt.Errorf("adding field event_type: %w", err)
+	}
+
+	cgroupIDField, err := containersDs.AddField("cgroup_id", api.Kind_Uint64)
+	if err != nil {
+		return nil, fmt.Errorf("adding field cgroup_id: %w", err)
+	}
+
+	mountNsIDField, err := containersDs.AddField("mntns_id", api.Kind_Uint64)
+	if err != nil {
+		return nil, fmt.Errorf("adding field mntns_id: %w", err)
+	}
```

Comment:
Not specifically related with this PR, but I think we should have a way to register a struct as a datasource to avoid having to manually add each field like this. Same for setting values.